evalml/tests/component_tests/test_components.py

```diff
@@ -949,6 +962,8 @@ def fit(self, X, y=None):
 def test_component_equality_all_components(component_class,
                                            logistic_regression_binary_pipeline_class,
                                            linear_regression_pipeline_class):
+    if component_class in {StackedEnsembleClassifier, StackedEnsembleRegressor}:
+        pytest.skip("Skipping stacked ensemble tests because pipelines not updated yet.")
```

Comment:
👍 

Please don't forget to update these in a later PR to the feature branch! I guess we'd notice the codecov drop on the feature branch PR, but, yeah, haha.

Perhaps when we're ready to merge the feature branch, we can do a quick before-vs-after diff of which tests passed vs were skipped. At that point the only difference in behavior should be from new tests or from tests which were deleted.